src/convertCSVtoSHP.py

```diff
@@ -0,0 +1,36 @@
+"""! @brief Converts CSVs to point SHPs."""
+
+"""@package src
+
+This module will process incoming travel episode CSVs 
+with fields Latitude, Longitude, and Time
+and convert them to point SHP files
+"""
+
+import pandas as pd
+import geopandas as gpd
+import os
+from os.path import join
+
+
+#This function receives a directory and returns an array of all files
+#within the directory, full file paths
+def readDir(inputpath):
+    return [os.path.join(inputpath, file) for file in os.listdir(inputpath)]
+
+
+#The function receives an array of csv epsiode files and 
```

Comment:
Please be more specific because its array of file paths

Reply:
updated, thanks!